demisto_sdk/commands/common/tests/git_tools_test.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
testing only the `legacy` tools, no?
consider renaming the file.

Reply:
done